src/main/frontend/components/file.cljs

```diff
@@ -152,7 +152,7 @@
        ;; wait for content load
        (and format
             (contains? (gp-config/text-formats) format))
-       (ui/loading "Loading ...")
```

Comment:
Looks like "..." was dropped off here and a couple other places. Was this done for consistency? Probably worth getting agreement from the authors if trying to make consistent or just leave it as it was before

Reply:
Yes, consistency was the main reason. I think `Loading...` is the best option. If the @logseq/core agrees, I will change the translation. If we want to keep both variations, I can introduce a loading-ellipsis key, but I don't think we should revert to the previous state.

Reply:
@xyhp915 @tiensonqin Do you care if we standardize on using `Loading` or `Loading...` for components? If not @sprocketc will change them to be consistently `Loading...`

Reply:
Second on `Loading...`.